rosbag2_py/src/rosbag2_py/_reader.cpp

```diff
@@ -35,61 +35,25 @@ namespace rosbag2_py
 {
 
 template<typename T>
-class Reader
+class Reader : public rosbag2_cpp::Reader
 {
 public:
   Reader()
-  : reader_(std::make_unique<rosbag2_cpp::Reader>(std::make_unique<T>()))
+  : rosbag2_cpp::Reader(std::make_unique<T>())
   {
   }
 
-  void open(
-    rosbag2_storage::StorageOptions & storage_options,
-    rosbag2_cpp::ConverterOptions & converter_options)
-  {
-    reader_->open(storage_options, converter_options);
-  }
-
-  bool has_next()
-  {
-    return reader_->has_next();
-  }
-
   /// Return a tuple containing the topic name, the serialized ROS message, and
   /// the timestamp.
-  pybind11::tuple read_next()
+  pybind11::tuple read_next_py()
```

Comment:
Makes sense - it hides the base class function. At first I avoided it because you can't overload a function with different return types, but this isn't actually an overload, it's a shadow. Changed it